pytorch/training/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -278,8 +277,9 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
 
 # Install smdebug from souce
 RUN cd /tmp \
-  && git clone -b ${SMDEBUG_VERSION} https://github.com/awslabs/sagemaker-debugger \
+  && git clone -b master --single-branch https://github.com/awslabs/sagemaker-debugger.git \
   && cd sagemaker-debugger \
+  && git checkout -f 9764a221fd3086ad8d2736128ccc2d69f269df1c \
```

Comment:
what commit is this? can we add an explanation?

Reply:
added comment, it is the numpy deprecation fix